src/styles/modal.css

```
@@ -0,0 +1,209 @@
+  }
+
+  .modal-description {
+    margin-left: 0;
+  }
+}
+
+@media (max-width: 800px) {
+  .modal {
+    width: 100%;
+
+    max-height: 80vh;
```

Stylelint (17.6.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)